backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -825,41 +825,40 @@ private void handleCancelledSystem() {
     private void handleFinalizingSuccess(final StateContext context) {
         log.info("Finalizing successful image transfer '{}' for {}", getCommandId(), getTransferDescription());
 
-        boolean isImageVerified = true;
+        ImageStatus nextImageStatus = ImageStatus.OK;
 
         // If stopping the session did not succeed, don't change the transfer state.
         if (stopImageTransferSession(context.entity)) {
             Guid transferingVdsId = context.entity.getVdsId();
 
-            // Verify image is relevant only on upload.
-            if (getParameters().getTransferType() == TransferType.Upload) {
-                isImageVerified = verifyImage(transferingVdsId);
-                if (isImageVerified) {
-                    // We want to use the transferring vds for image actions for having a coherent log when transferring.
-                    setVolumeLegalityInStorage(LEGAL_IMAGE);
-                    if (getDiskImage().getVolumeFormat().equals(VolumeFormat.COW)) {
-                        setQcowCompat(getDiskImage().getImage(),
-                                getStoragePool().getId(),
-                                getDiskImage().getId(),
-                                getDiskImage().getImageId(),
-                                getStorageDomainId(),
-                                transferingVdsId);
-                        imageDao.update(getDiskImage().getImage());
-                    }
+            // Verify image is relevant only on upload
```

Comment:
Suggest to take this comment to be a few line below to be after `else if (verifyImage(transferingVdsId)) {`.
Add dot at the end.

Reply:
It doesn't add enough value in my opinion, I'd rather not add changes that aren't necessary since this patch may need to be backported